extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -216,9 +227,7 @@ function SidebarWithRef(
           size={false}
           layout={true}
           paint={true}
-          className={`${classes.sidebarClass} ${
-            side === 'left' ? classes.left : classes.right
-          }`}
+          className={`${classes.sidebarClass}`}
```

Comment:
Don't need template literal since we are providing one class only:
```suggestion
          className={classes.sidebarClass}
```

or take and passthrough `className` declared in the component props (probably preferred?) though it looks like this currently gets overwritten due to use of `...rest` - is this on purpose?
```suggestion
          className={`${props.className} ${classes.sidebarClass}`}
```

Reply:
Thanks removed template literals and updated to use `wrapperClassName` so `className` will still propagate through.